tests/Collector/BoolCollectorTest.php

```diff
@@ -48,13 +48,13 @@ public function getCalculatorMock($returns)
         return $collector->reveal();
     }
 
-    public function provideStatisfyBasic()
+    public function provideSatisfyBasic()
     {
         # must
         yield [
             [
                 'must' => [
-                    ['type' => true],
+                    ['type' => 'true'],
```

Comment:
why strings?

Reply:
https://github.com/sensiolabs-de/deptrac/blob/master/src/Collector/CollectorInterface.php#L13-L19 says it should be a string and the value is used as an array key at https://github.com/sensiolabs-de/deptrac/blob/master/src/CollectorFactory.php#L16

Reply:
well, it's ok even if i think it's a bit overkill. `getCollectorMock('true', true)` and `'true'` is brainfuck.
i'd prefer casting the bool to a string. there will never be other values than true / false.

Reply:
truly, mixing strings and booleans (true / false) is not a nice style. we're talking about the type here, and the type has a name. "true" or "false" as a string. Or "boolean", "object", "null" etc. And not if the state is true or false.

Reply:
@timglabisch I added some more improvements and fixed the "brainfuck" as you mentioned. Even I could apply the validity with phpstan level 7.